style.css

```
@@ -0,0 +1,54 @@
+body{
+    margin: 0;
+    padding: 0;
+    background: #262626;
+}
+.cup{
+    position: absolute;
+    top: 50%;
+    right: 50%;
+    transform: translateY(-50%,-50%); 
```

```suggestion
    left: 50%;
    transform: translate(-50%,-50%); 
```

if want to center a div then you should do this...